sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -299,4 +311,22 @@ public boolean isAllowedGroup(String group) {
                        .orElseGet(Collections::emptyList)
                        .contains(group);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
```

Comment:
Please move all these logics in to `afterPropertiesSet`:
1. Set default value. For `baseUri`, `graphBaseUri` and `graphMembershipUri`.
2. `addSlash`. For both `baseUri` and `graphBaseUri`.
3. Validate values for `baseUri` and `graphBaseUri`.